Keas.Mvc/ClientApp/components/Access/AccessEditValues.tsx

```diff
@@ -71,7 +71,7 @@ export default class AccessEditValues extends React.Component<IProps, {}> {
                 )}
                 <div className="wrapperasset">
                     <FormGroup>
-                        <Label for="item">Item</Label>
+                        <Label for="name">Name</Label>
```

Comment:
the `for="name"` doesn't actually do anything unless we added a name attr to the related input, which maybe we should do.  I think it's an accessibility thing.  Nothing to do with your PR really, just something that stuck out